WKData/Sources/WKData/Models/Shared/WKEditSummaryTag.swift

```diff
@@ -12,4 +12,5 @@ public enum WKEditSummaryTag: String {
     case diffRollback = "#diff-rollback"
     case talkReply = "#talk-reply"
     case talkTopic = "#talk-topic"
+    case suggestedEditsAddImageTop = "#suggestededit-add-image-top"
```

Comment:
This was taken from Android's [codebase](https://github.com/wikimedia/apps-android-wikipedia/blob/807850511c22a73fe6620257e616ce5e82a81c21/app/src/main/java/org/wikipedia/suggestededits/SuggestedEditsImageRecsFragment.kt#L465).